src/components/Menu.js

```diff
@@ -327,8 +327,9 @@ export type MenuListProps = {
 export type MenuListComponentProps = CommonProps &
   MenuListProps &
   MenuListState;
-export const menuListCSS = () => ({
+export const menuListCSS = ({ maxHeight }: MenuListComponentProps) => ({
   flexGrow: 1,
+  flexBasis: (maxHeight - spacing.baseUnit*2),
```

Comment:
In IE11 flex-basis auto is ignored. We need to specify an initial size for our flex element, otherwise our content will overflow. 

Luckily we have access to the maxHeight of our menu.